config/sets/symfony/symfony7/symfony74.php

```diff
@@ -9,4 +9,5 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/symfony74/symfony74-console.php');
     $rectorConfig->import(__DIR__ . '/symfony74/symfony74-framework-bundle.php');
+    $rectorConfig->import(__DIR__ . '/symfony74/symfony74-routing.php');
```

Comment:
Please register to Symfony7SetProvider as well

https://github.com/rectorphp/rector-symfony/blob/8f7905fe0d2acb163a306933a4c9c302c15cf9cf/src/Set/SetProvider/Symfony7SetProvider.php#L12

Reply:
Done